src/main/java/seedu/address/ui/WidgetViewBox.java

```diff
@@ -132,17 +134,19 @@ public void setToDefaultView() {
         widgetScrollPane.setVbarPolicy(ScrollPane.ScrollBarPolicy.NEVER);
         tbmLogoContainer.getChildren().clear();
         tbmLogoContainer.getChildren().add(tbmLogo);
-        tbmLogoContainer.getChildren().add(timer);
         tbmLogoContainer.setStyle("-fx-padding: 20 0 25 0; -fx-spacing: 10");
-        textClock = new TextClock(timer);
+        if (textClock != null) {
```

Comment:
is this check needed since textClock would be reinitialised anyway and subsequently call `textClock.play()`?

Reply:
Yup, this check is needed since the very first time the WidgetViewBox is initialized, the textClock is null, and we call this function